clean.nf

```diff
@@ -1,10 +1,25 @@
 #!/usr/bin/env nextflow
 
-if( !nextflow.version.matches('20.01+') ) {
-    println "This workflow requires Nextflow version 20.01 or greater -- You are running version $nextflow.version"
-    exit 1
+/* 
+Nextflow version check  
```

Comment:
I just found out that you can set the nextflow version in the manifest scope in the config: https://www.nextflow.io/docs/latest/config.html#nextflow-version

This would only print a warning:
```
WARN: Nextflow version 20.10.0 does not match workflow required version: >=21.04.0 -- Execution will continue, but things may break!
```

Do you think a warning is enough or wuold you rather stick to a `exit 1`?

Reply:
uhh thats actually quite nice! I would then go for the warning rather the exit! Thanks!